torch/csrc/distributed/c10d/TraceUtils.h

```diff
@@ -439,6 +456,7 @@ struct NCCLTraceBuffer {
       if (completed) {
         r.state_ = "completed";
       }
+      r.duration_ = getDurationFromFirstEvent(*r.start_, *r.end_);
```

Comment:
oops, i should probably put the getDuration up above in the if(completed) clause.

actually, does anyone know if its valid to measure the duration of a collective that crashed or was aborted?  cc @kwen2501